assets/scripts/aircraft.js

```diff
@@ -787,13 +858,16 @@ var Aircraft=Fiber.extend(function() {
       return ['ok', 'turn ' + direction + 'heading ' + heading_to_string(this.requested.heading), ''];
     },
     runAltitude: function(data) {
-      var split     = data.split(" ");
+      if(data[0] == "\u2B61" || data[0] == "\u2B63") {  //shortKey 'v' or '^' in use
```

Comment:
If the COMMANDS data is updated with a different key for altitude then this will break.  Would it be possible to strip the shortKey off in the parsing step and just pass the heading data?